src/cameras/CubeCamera.js

```diff
@@ -82,3 +83,14 @@ THREE.CubeCamera = function ( near, far, cubeResolution ) {
 
 THREE.CubeCamera.prototype = Object.create( THREE.Object3D.prototype );
 THREE.CubeCamera.prototype.constructor = THREE.CubeCamera;
+
+THREE.CubeCamera.prototype.clone = function () {
+
+	var camera = new THREE.CubeCamera();
+
+	THREE.Object3D.prototype.clone.call( this, camera, false );
+	this.renderTarget.clone(camera.renderTarget);
```

Comment:
I am not sure a clone method is needed here. But even if @mrdoob wants it, I do not believe this code will clone the children properly.